lib/std/posix.zig

```diff
@@ -7254,8 +7256,8 @@ pub const TimerFdCreateError = error{
 pub const TimerFdGetError = error{InvalidHandle} || UnexpectedError;
 pub const TimerFdSetError = TimerFdGetError || error{Canceled};
 
-pub fn timerfd_create(clock_id: clockid_t, flags: system.TFD) TimerFdCreateError!fd_t {
-    const rc = system.timerfd_create(clock_id, @bitCast(flags));
+pub fn timerfd_create(clockid: system.timerfd_clockid_t, flags: system.TFD) TimerFdCreateError!fd_t {
```

Comment:
Likewise, the rename from `clock_id` to `clockid` shouldn't be necessary anymore.

Reply:
yep